includes/class-ckwc-integration.php

```diff
@@ -135,8 +171,55 @@ public function init_form_fields() {
 				'class'       => 'enabled subscribe',
 			),
 
+			// Custom Field Mappings.
+			'custom_field_phone'            => array(
+				'title'       => __( 'Send Phone Number', 'woocommerce-convertkit' ),
+				'type'        => 'custom_field',
+				'default'     => '',
+				'description' => __( 'The ConvertKit Custom Field to store the Order\'s Phone Number.', 'woocommerce-convertkit' ),
```

Comment:
Nitpick: we don't use [title case](https://en.wikipedia.org/wiki/Title_case), so only ConvertKit should be capitalized: